CHANGELOG.unreleased.md

```diff
@@ -20,6 +20,7 @@
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
 - [Backup] Detect and clear orphan merge states, fix `ENOENT` errors (PR [#6087](https://github.com/vatesfr/xen-orchestra/pull/6087))
+- [Backup] Ensure merges are also executed after backup on S3 (PR [#6095](https://github.com/vatesfr/xen-orchestra/pull/6095)), maintaining the size of the vhd size under control
```

Comment:
Format is weird, the PR mention should be at the end.

Any refs for this PR? (issues, forum, etc)

Reply:
that's from https://xcp-ng.org/forum/topic/5256/s3-backup-try-it/119 , but I don't know how to target a specific post

Reply:
Right-click on the post time to get the URL of the specific post